command_impl.go

```diff
@@ -23,6 +31,20 @@ func (cmd *commandImpl) onError(err error) {
 	}
 }
 
+func (cmd *commandImpl) getName(n string) string {
+	if n == "" {
+		panic("getName: n must not be empty")
+	}
+	if cmd.name == "" {
+		if EnableDebugLogging == true {
+			cmd.name = fmt.Sprintf("%s-%v", n, atomic.AddUint64(&c, 1))
```

Comment:
This will show executed commands with a unique suffix in the debug output which makes it easier to correlate log lines.